Dockerfile/Dockerfile

```diff
@@ -11,28 +11,23 @@ RUN apt-get update \
     curl \
   && apt-get clean
 
-RUN sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' && \
-  wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add - && \
+RUN wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add - && \
```

Comment:
We should use packages.ros.org for Edifice + Galactic.

At the moment, we only need packages.osrfoundation.org if we want to test combinations that include Fortress, because that's not on packages.ros.org yet (see https://github.com/ros-infrastructure/reprepro-updater/pull/128). In that case, I'd suggest we conditionally include the OSRF packages just when testing Fortress, so we don't use a version that's not on the ROS farm by mistake.

This is the problem I'd like to avoid:

1. We release a new feature into `ign-gazebo` 5.X (Edifice) on packages.osrfoundation.org (but not yet on packages.ros.org)
2. A PR to this repo uses that feature, passes CI, and gets merged
3. This results in:
    * Users building this from source against packages.ros.org are broken
    * Trying a new bloom release of this package into the ROS farm will fail